bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -43,9 +43,21 @@
         mSavedStateHandler = savedStateHandler;
     }
 
+    void clear(@NonNull Object target) {
+        String uuid = mObjectUuidMap.get(target);
+        if (uuid == null) {
+            return;
+        }
+        clearDataFromDisk(uuid);
```

Comment:
Do you also want to remove it from `mObjectUuidMap` once it's cleared?  Or is it better/necessary to just keep that UUID around?